android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -827,6 +1057,7 @@ public void createDataChannel(final int peerConnectionId, String label, Readable
             if (-1 != dataChannelId) {
                 dataChannel.registerObserver(new DataChannelObserver(dataChannelId, dataChannel));
                 mDataChannels.put(dataChannelId, dataChannel);
+                registerDataChannelObserver(dataChannelId, dataChannel);
             }
```

Comment:
@zxcpoiu, the mistake must be mine made during merging upstream into the fork. I'll look into it ASAP.